src/Components/Main/Pages/ProfilePage/PostForm/PostFormContainer.js

```diff
@@ -0,0 +1,29 @@
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { PostForm } from './PostForm';
+
+const validationSchema = Yup.object().shape({
+  text: Yup.string()
+    .max(100, 'Must be shorter than 100 characters')
+    .required('Required')
+});
+
+export const PostFormContainer = ({addPost}) => {
+  const onSubmit = (formData, {setSubmitting, resetForm}) => {
+    setSubmitting(false);
```

Comment:
Should it be  setSubmitting(true);? =)

Reply:
I used setSubmitting(false) because I specified the attribute disabled={isSubmitting} in the button, and setSubmitting sets as true by default.
Redundant logic for such case is corrected.